src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -755,19 +757,68 @@ object TezosDatabaseOperations extends LazyLogging {
   def writeProcessedEventsLevels(eventType: String, levels: List[BigDecimal]): DBIO[Option[Int]] =
     Tables.ProcessedChainEvents ++= levels.map(Tables.ProcessedChainEventsRow(_, eventType))
 
-  /** Inserts registered tokens to the table if empty
-    *
+  /** Inserts to the table if table is empty
+    * @param table slick TableQuery[_] to which we want to insert
     * @param rows the type of event to record
```

Comment:
"event" here might be a copy/paste error

Reply:
yes, changed